src/applications/widget-editor/src/components/widget-editor/component.js

```diff
@@ -25,8 +26,16 @@ const WidgetEditor = ({
     throw new Error("Widget editor: Missing prop datasetId of type string");
   }
 
+  if (!map?.mapboxToken) {
```

Comment:
Shouldn't it be?
```suggestion
  if (!!map && !map.mapboxToken) {
```

I'm thinking that if the map is disabled, we shouldn't ask for a token, no?